default.json

```diff
@@ -7,7 +7,8 @@
     ":configMigration",
     ":pinAllExceptPeerDependencies",
     "github>reside-eng/renovate-config:private-npm",
-    "github>reside-eng/renovate-config:actions-ci"
+    "github>reside-eng/renovate-config:actions-ci",
+    "customManagers:biomeVersions"
```

Comment:
Can you move this before our two custom ones above? Just so we have the ability to override.